client/packages/common/src/ui/icons/FiberManualRecord.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const FiberManualRecordIcon = (props: SvgIconProps): JSX.Element => {
```

Comment:
Think in the context where it is used we should rename it like CircleCheckMarkIcon or similar...

I guess it comes a set of Record, Pause, Play icons?

Reply:
we already have `Circle.tsx` can you please use that one 🙏